packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3399,6 +3401,24 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
       markNeedsSemanticsUpdate();
   }
 
+  /// The handler for [SemanticsActio.setSelection].
+  ///
+  /// This handler is invoked when the user either wants to change the currently
+  /// selected text in a text field or change the position of the cursor.
+  ///
+  /// TalkBack users can trigger this handler by selecting "Move cursor to
+  /// beginning/end" or "Select all" from the Local Context Menu.
```

Comment:
nit: why capitalize "local context menu"?

Reply:
Changed to lower-case.